runtime/lua/vim/lsp/buf.lua

```diff
@@ -30,7 +30,7 @@ end
 ---
 ---@param method (string) LSP method name
 ---@param params (optional, table) Parameters to send to the server
----@param handler (optional, functionnil) See |lsp-handler|. Follows |lsp-handler-resolution|
+---@param handler (optional, function) See |lsp-handler|. Follows |lsp-handler-resolution|
```

Comment:
I assume these changes are left over, I'm fine with them ofc but it should be collected into a bigger style cleanup PR